pkg/clusteragent/admission/mutate/config/config_test.go

```diff
@@ -501,11 +500,12 @@ func BenchmarkJSONPatch(b *testing.B) {
 
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		request := admission.MutateRequest{
+		request := admission.Request{
 			Raw:       podJSON,
 			Namespace: "bar",
 		}
-		jsonPatch, err := webhook.MutateFunc()(&request)
+		admissionResponse := webhook.WebhookFunc()(&request)
+		jsonPatch, err := json.Marshal(admissionResponse.Patch)
 		if err != nil {
 			b.Fatal(err)
 		}
```

Comment:
FWIW, `testing.B` should implement the interface expected by the `testify` module so that this should work as well:
```suggestion
		require.NoError(b, err)
```

It might be more CPU expensive than the `if err != nil`, but I suspect the cost of `require.NoError(…)` to remain negligible compared to the invocation of the hook and the JSON marshaling so that it shouldn’t impact the outcome of the benchmark.